app/controllers/application_controller.rb

```diff
@@ -25,7 +25,7 @@ def check_staging_auth
   end
 
   def authenticate?
-    !(Rails.env.development? || Rails.env.test?)
+    !(Rails.env.production? || Rails.env.development? || Rails.env.test?)
```

Comment:
is there any value in having this then? And if it's just for staging can we change it to `Rails.env.staging?`